data/governments.txt

```diff
@@ -1178,6 +1222,51 @@ government "Quarg (Hai)"
 		ship "Quarg Skylark"
 		ship "Quarg Wardragon"
 
+government "Quarg (Incipias)"
+	swizzle 0
+	color .88 .77 0
+	"display name" "Quarg"
+
+	"player reputation" 1
+	"attitude toward"
+		"Quarg" 1
+		"Quarg (Hai)" 1
+		"Quarg (Kor Efret)" 1
+		"Quarg (Gegno)" 1
+		"Merchant" .01
+		"Hai" .01
+		"Hai (Wormhole Access)" .01
+		"Pirate" -.01
+		"Pirate (Devil-Run Gang)" -.01
+	"custom penalties for"
+		"Conlatio"
+			disable -10.
+			board -10.
+			capture -20.
+			destroy -40.
```

Comment:
```suggestion
			disable 10
			board 10
			capture 20
			destroy 40
```
Negative penalty for an action means you _gain_ reputation for doing it, which presumably isn't what you intend here. Also, we don't use . after integers in custom penalties.